master/internal/api_user_intg_test.go

```diff
@@ -52,6 +52,10 @@ func setupAPITest(t *testing.T, pgdb *db.PgDB) (*apiServer, model.User, context.
 			system = actor.NewSystem("mock")
 			ref, _ := system.ActorOf(sproto.K8sRMAddr, actor.ActorFunc(
 				func(context *actor.Context) error {
+					switch context.Message().(type) {
```

Comment:
Are the changes to this file supposed to be part of the PR?

Reply:
Yeah, the experiment deletion flow needs the resource manager to respond to ```sproto.DeleteJob``` and calls something in ```trialLogBackend```